edn/src/symbols.rs

```diff
@@ -98,12 +99,73 @@ impl Keyword {
 }
 
 impl NamespacedKeyword {
+    /// Creates a new `NamespacedKeyword`. When `name` starts with a `_`, it is considered
```

Comment:
I'm not super happy with 'straight'. Quite apart from the exclusionary language, this isn't a thing that bends; it's a directional flag where the two directions are opposite. So `reversed`, or `direction: forwards/backwards`, perhaps even `inverse`, but not `straight`.